Resources/Prototypes/Objectives/traitor.yml

```diff
@@ -111,6 +111,20 @@
     # if ce flies a shittle to centcom you better find a way onto it
     requireDead: true
 
+- type: entity
+  parent: [BaseTraitorObjective, BaseKillObjective]
+  id: KillRandomTraitorObjective
+  description: We have reason to believe that they have become unloyal to the syndicate and need them eliminated.
```

Comment:
The word "unloyal" is not standard usage. While it is widely used, "disloyal" is the correct term. Up to you if you want to change it or not.

Reply:
I just decided to change the wording entirely! I didn't realize that was such an uncommon word.